core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -417,23 +324,7 @@ trait EntityComponent {
       }
 
       // generate the clause to filter based on user search terms
-      def paginationFilterSql(prefix: String, alias: String, entityQuery: model.EntityQuery) = {
-        val filtersOption = entityQuery.filterTerms.map {
-          _.split(" ").toSeq.map { term =>
-            sql"concat(#$alias.name, ' ', #$alias.all_attribute_values) like ${'%' + term.toLowerCase + '%'}"
-          }
-        }
-
-        filtersOption match {
-          case None => sql""
-          case Some(filters) =>
-            concatSqlActions(
-              sql"#$prefix (",
-              reduceSqlActionsWithDelim(filters, sql" #${FilterOperators.toSql(entityQuery.filterOperator)} "),
-              sql") "
-            )
-        }
-      }
+      def paginationFilterSql(prefix: String, alias: String, entityQuery: model.EntityQuery) = sql""
```

Comment:
this removes the ability to search within the `all_attribute_values` column - but that's only used by non-Quicksilver workspaces, so its ok to delete